### ISSUE TITLE
some questions about the environment

### ISSUE THREAD

**Issue by MingShyanWei** [yagguc/deep_impression#2]
Excuse me, I have some questions about the environment.
Trying to restore your experiment results now.
But I have some problems.
I would like to ask about the setting of the original environment.
python version
cupy, chainer, librosa, numpy, skvideo version

Thank you very much.

**Comment by shabazbelim** [comment]
Hi, I am trying to evaluate your results but not getting the same accuracy as mentioned in the paper. Do you have any repository of training code?

**Comment by aircrushin** [comment]
Hi, same problem there. have you guys solved it?
